internal/host/host_test.go

```diff
@@ -607,6 +607,52 @@ var _ = Describe("reset host", func() {
 
 })
 
+var _ = Describe("register host", func() {
+	var (
+		ctx           = context.Background()
+		db            *gorm.DB
+		state         API
+		h             models.Host
+		eventsHandler events.Handler
+		dbName        = "reset_host"
+		config        Config
+	)
+
+	BeforeEach(func() {
+		db = common.PrepareTestDB(dbName, &events.Event{})
+		eventsHandler = events.New(db, logrus.New())
+		config = *defaultConfig
+		dummy := &leader.DummyElector{}
+		state = NewManager(getTestLog(), db, eventsHandler, nil, nil, nil, nil, &config, dummy)
+	})
+	AfterEach(func() {
+		common.DeleteTestDB(db, dbName)
+	})
+
+	It("register host success", func() {
+		id := strfmt.UUID(uuid.New().String())
```

Comment:
please move common code into before each, for example creation if the cluster and the id.